src/Common/src/Interop/Windows/user32/Interop.GetWindowText.cs

```diff
@@ -10,7 +10,7 @@ internal partial class Interop
 {
     internal partial class User32
     {
-        [DllImport(Libraries.User32)]
+        [DllImport(Libraries.User32, EntryPoint = "GetWindowTextW")]
```

Comment:
> Today we only run the UAPAOT tests on x64 which is why I didn't see if this was a problem

Though I'd have expected this to be an issue on 64-bit as well, as I thought the default was always Ansi regardless of platform.